lib/releaselib.sh

```diff
@@ -620,6 +621,9 @@ release::gcs::publish_version () {
                  $type-$version_major
                  $type-$version_major.$version_minor
                 )
+  if [[ -n "${build_alias:-}" ]]; then
```

Comment:
you can just add $local_publish_file to the array above.  It'll only add to the array if it's set.